kr-core/src/main/clojure/edu/ucdenver/ccp/kr/sparql.clj

```diff
@@ -404,14 +466,23 @@
      (apply str (interleave (map sparql-query-body triple-pattern)
                             ;;(repeat ". \n")))
                             (repeat " \n")))
-   (sparql-operator? (first triple-pattern)) (filter-body triple-pattern)
+   ;; used as a catch-all; this routes anything unusual to a FILTER.
+   ;; (sparql-operator? (first triple-pattern)) (filter-body triple-pattern)
```

Comment:
This change is going to break backwards compatibility.  Before operators were implicitly wrapped in a filter, whereas now the filter body must be explicitly specified.

I really should have seen this earlier  ):